README.md

```diff
@@ -172,14 +194,20 @@ Strict equality checks (`===`) must be used in favor of abstract equality checks
 undefOrNull == null;
 ```
 
+When checking for `undefined`, prefer using `void(0)` instead.
```

Comment:
Not sure what's the advantage here. In ES5+, the global `undefined` is immutable and IMO `undefined` is much more readable than `void 0`.